db-service/lib/SQLService.js

```diff
@@ -170,7 +170,7 @@ class SQLService extends DatabaseService {
       // REVISIT: this is a hack the target of $now might not be a timestamp or date time
       // Add input converter to CURRENT_TIMESTAMP inside views using $now
       if(/^CREATE VIEW.* CURRENT_TIMESTAMP[( ]/is.test(query)) {
```

Comment:
```suggestion
      if(this.options.kind === 'sqlite' && /^CREATE VIEW.* CURRENT_TIMESTAMP[( ]/is.test(query)) {
```
i need this locally to not apply the patch during deployment to postgres... why hasn't anybody else run into this?

Reply:
I guess we don't test the replacement yet.

Reply:
I've double checked and the function `onPlainSQL` is overridden in postgres.

Reply:
then we should probably check why deploy to postgres reached there...